plugins/woocommerce/client/admin/client/marketplace/components/marketplace-link/marketplace-link.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { Text } from '@woocommerce/experimental';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Link } from '@woocommerce/components';
+import { getAdminLink } from '@woocommerce/settings';
+import { recordEvent } from '@woocommerce/tracks';
+
+interface MarketplaceLinkProps {
+	/**
+	 * HTML element to use for the Text component
+	 */
+	as?: string;
```

Comment:
I'm assuming this prop name is defined by child components such as `Text`? Shame, I find this very confusing so I appreciate the included comment!

I think `element` or `elementType` would have been better.